clab/config.go

```diff
@@ -79,12 +80,21 @@ func (c *CLab) parseTopology() error {
 	if c.Config.ConfigPath == "" {
 		c.Config.ConfigPath, _ = filepath.Abs(os.Getenv("PWD"))
 	}
+	if c.Config.Prefix == nil {
+		c.Config.Prefix = new(string)
+		*c.Config.Prefix = defaultPrefix
```

Comment:
can this be 

```go
c.Config.Prefix = &defaultPrefix
```

https://replit.com/join/nwneuvdywh-romandodin

Reply:
that's possible only if you change `defaultPrefix` from const to a variable

Reply:
ah, roger...